plugins/scaffolder-actions/scaffolder-backend-module-utils/package.json

```diff
@@ -38,7 +38,7 @@
     "@backstage/plugin-scaffolder-node": "^0.2.8",
     "adm-zip": "^0.5.9",
     "cross-fetch": "^3.1.4",
-    "detect-indent": "^7.0.1",
+    "detect-indent": "^6.1.0",
```

Comment:
oh, sorry. We probably need the modifications to the yarn.lock file into this patch set also